src/pylhe/__init__.py

```diff
@@ -20,6 +20,9 @@ def __init__(self, eventinfo, particles, weights=None, attributes=None):
         for p in self.particles:
             p.event = self
 
+    def visualize(self, outputname):
+        visualize(self, outputname)
```

Comment:
What is this duplication of `visualize`?

Reply:
Otherwise all looks good, modulo the change pypdt to particle in setup.cfg.

Reply:
> What is this duplication of `visualize`?

@fuenfundachtzig can verify, but I assume it is because at the moment all the code is in this single `__init__.py` file that to give the `LHEfile` a `visualize` method they didn't need to import `visualize`.

This is a bit awkward looking, but assuming I'm not missing something else this could get cleaned up more down the road with some refactoring of the library structure.

Reply:
Correct.